web/js/date/wv.date.js

```diff
@@ -1,39 +1,35 @@
-var wv = wv || {};
+import util from '../util/util';
```

Comment:
Jo and I's branch is written like this `import util from '../util/util.js';` with `.js`. added but I think dropping the .js looks cleaner like you have it.